pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -102,6 +104,13 @@ stdenv.mkDerivation rec {
   ] ++ optionals gtkSupport [
     # for gtksink
     gtk3
+  ] ++ optionals qt5Support (with qt5; [
```

Comment:
I don't think you want to input qt5 directly but use libforQt5 in callPackages.

Reply:
Are there docs on that?

Reply:
I'm not sure it's really needed... `libsForQt5.callPackage` ensures all the dependencies are using the same Qt version, which is significant for a Qt application but this is a library and I don't think any other dependency depends on Qt.

Reply:
relevant documentation https://nixos.org/manual/nixpkgs/stable/#sec-language-qt